whole-tale/girder_wholetale
# Add an endpoint that generates the Tale manifest file. Fixes #250

This PR adds an endpoiont to the existing Tale endpoint to generate the contents of the proposed Tale manifest file. Related to #250 

To Test
   1. Check out this branch
   2. Create a Tale with a workspace and or data sets
   3. Navigate to the Girder API page
   4.  Scroll down to /tale/{id}/manifest 
   5. Paste your Tale ID and a license SPDX 
   6. Click `Try it out!`
